docker-compose.yml

```diff
@@ -7,22 +7,24 @@ services:
       context: .
       target: development
     environment:
+      # adjust these vars in production!
       DATABASE_USER: postgres
       DATABASE_USER_PASSWORD: postgres
       DATABASE_NAME: mindwendel
       DATABASE_PORT: 5432
       DATABASE_HOST: db
-      # prod related env vars
-      # SECRET_KEY_BASE:
-      # URL_HOST: localhost
+      DATABASE_SSL: "false"
+      MW_DEFAULT_LOCALE: de
+      SECRET_KEY_BASE: 24Ip+GncLkNy+EAyC5wG9K5zIHNBmeLa6d87bhabAueuvHUy34Qfjs41+zWKDrJm
```

Comment:
i would remove this, ppl will just use it for prod purposes which presents a security risk. Just add a comment to fill it :)